Assets/Plugins/Mediabox/Source/GameKit/GameManager/PauseHandler.cs

```diff
@@ -2,30 +2,58 @@
 
 namespace Mediabox.GameKit.GameManager {
     public class PauseHandler {
-        float timeScaleBeforePause;
-        float volumeBeforePause;
-        bool isPaused;
+        float timeScaleBeforePause = 1;
+        float volumeBeforePause = 1;
```

Comment:
These are actually in case someone were to use a subscribed pauseEvent, and somehow removed the subscription before the unpause. In that case no value for the timescalebeforepause would be recorded, and the default being 0, would mean that the game would "unpause" to timescale 0.

Reply:
(reiterating a note from above, if we move the Pause into the Game base. Then I think we need to initialize these values in each game, as the memory for them will be lost as the games unload and then load anew.)